### ISSUE TITLE
{ju'anai} is not a lujvo.

### ISSUE THREAD

**Issue by lagleki** [lojban/vlasisku#2]
http://vlasisku.lojban.org/vlasisku/ju'anai says `Components in this compound:`. Clearly, the components should not be {jufra} and {nabmi}.

**Comment by mantycore** [comment]
I guess the part of issue is what vlasisku do not check for double consonats cluster in the first five letters of lujvo, maybe? http://vlasisku.lojban.org/vlasisku/be%27ebe%27e

**Comment by durka** [comment]
Correct, the lujvo decomposer function that vlasisku uses creates many
false positives because it doesn't check for required hyphens. The version
in my fork already uses the parser to break apart lujvo so this issue is
solved (though my version doesn't break up cmavo compounds, it just says
not found, maybe I should add that).

**Comment by lagleki** [comment]
should be just using ilmentufa to split words.

2015-02-15 8:56 GMT+03:00 Alex Burka <notifications@github.com>:

> Correct, the lujvo decomposer function that vlasisku uses creates many
> false positives because it doesn't check for required hyphens. The version
> in my fork already uses the parser to break apart lujvo so this issue is
> solved (though my version doesn't break up cmavo compounds, it just says
> not found, maybe I should add that).
>

**Comment by lagleki** [comment]
or la cmaxes which is a purely morphological parser http://mw.lojban.org/extensions/ilmentufa/cmaxes.html

**Comment by Pendrokar** [comment]
Seems fixed.
